prow/manifests/overlays/metal3/config.yaml

```diff
@@ -2735,7 +2735,16 @@ presubmits:
     agent: jenkins
     always_run: false
     optional: true
-
+  # {job_prefix}-ironic-image-build-test
+  - name: metal3-ironic-image-build-test
+    branches:
+    - main
+    - release-23.1
+    - release-24.0
+    - release-24.1
```

Comment:
Add also 25.0 branch, or better yet, since if this is expected to be run on all branches, leave the branches config out.